app/services/cards_service.rb

```diff
@@ -0,0 +1,39 @@
+class CardsService
+  class << self
+    def parsing_html(user_id, block_id, url, css_original, css_translated)
+      parse = false
+
+      if block_id && user_id
+        begin
+          html = Nokogiri::HTML(open(url))
+        rescue Exception
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?